Nitrox.Test/Patcher/Patches/Dynamic/InfectionReveal_PatchTest.cs

```diff
@@ -0,0 +1,15 @@
+using HarmonyLib;
+using NitroxTest.Patcher;
+
+namespace NitroxPatcher.Patches.Dynamic;
+[TestClass]
+public class InfectionReveal_PatchTest
+{
+    [TestMethod]
+    public void InjectionSanity()
+    {
+        IEnumerable<CodeInstruction> beforeInstructions = PatchTestHelper.GetInstructionsFromMethod(InfectionReveal_Patch.TARGET_METHOD);
+        IEnumerable<CodeInstruction> result = InfectionReveal_Patch.Transpiler(beforeInstructions);
+        Assert.AreEqual(beforeInstructions.Count() + 1, result.Count());
+    }
+}
```

Comment:
Should use new way of testing patches. See `PatchesTranspilerTest` and add your `InfectionReveal_Patch` to the list.

Reply:
Fortunately master was a clean merge into my branch since PatchesTranspilerTest was made after I diverted, implemented now and test passes.